### ISSUE TITLE
[Feature]: --shared option could allow to share multiple folders, not tmp folder only

### ISSUE THREAD

**Issue by JanuszChmiel** [termux/proot-distro#292]
### Feature description

Dear elite proot and proot-distro developers.
How complex would be to implement this feature?
So user would have The chance to define shared folder not only TMP, but also home?
The aim of this plea is to speed up Speech-dispatcher speech server responsiveness because.
Orca which would run on Proot container would communicate with Speech-dispatcher which run from Termux without Proot.
It would speed up The responsiveness of The Orca screen reader app.

Because if speech-dispatcher must cooperate with Proot and must call many .so libraryes on Prooted container, it is long delay.

So what would happen, if also Home directory would be shared?
Speech-dispatcher always creates its socked in The home folder.

**Comment by JanuszChmiel** [comment]
Oh, I have been overjoyed too prematurely.
Because simply typing
proot-distro login archlinux --bind $HOME:$HOME
Do not allow spd-say command to automatically connect to The speech-dispatcher socket. Because sockets are created in The proot-distro Archlinux directory hiearchi related to user name.
So as I Am analysing and as I Am thinkink deeply, The best option will be to convince elite developers to build Orca directly for Termux. Speech-dispatcher is allready compiled for Termux including spd-say command which works.
But thank you for your attempt to help Me. Your advice have been pperfect. But unfortunately for Me, Linux is much more complex as I would wanted for Orca screen reader and its interaction with The Speech-dispatcher speech server.

**Comment by JanuszChmiel** [comment]
Please, if I type
proot-distro login archlinux  --termux-home 
It do not functioning.
How to use --termux-home  option when starting Linux distribution while usingproot-distro?
I have found out, that Proot-distro provided chroots are best configured for Termux and Proot. And Proot-distro startup script are always introducing new Proot options for startup if Proot developers deelop new Proot feature.
Including shared memory support, /proc folder emulation, Etc.
Thank you very much dear MR
sylirre

**Comment by JanuszChmiel** [comment]
Oh you have solved my issue very weel done!!!
I Am able to start ArchLinux or even The Fedora 37 distribution with Speech-dispatcher speech server compiled for Termux.
So Speech-dispatcher compiled for Fedora or ArchLinux is not being used. To have The 100 ensurance, I have removed compiled Speech-dispatcher binary module from /usr/sbin
And really.
spd-say and even Orca complex screen reader really communicate with The Speech-dispatcher speech server from Termux.
No crashes are occurring.
And very fast responsiveness, because this technique bypasses The necessarity to use Proot for executing Speech-dispatcher specific compiled speech server for An Linux distribution.
And because I have found Termux-x11 client app very very useful, I Am running Linux distributions by typing
proot-distro login --user janusz --shared-tmp --termux-home archlinux

So thank you very much. I have been afraid, that I will have to recode Orca complex Python script. Fortunately, Linux sockets are perfectly identified automatically thanks to very useful proot-distro command line options.
You have helped many potential severely visually impaired Orca users who would like to use fast speech rtesponsiveness with this assistive technology.

**Comment by JanuszChmiel** [comment]
But I have found out, that user must be careful when running Linux distribution byh this way. Best practice is to use only one Linux distribution, not switch between various ones.Because even ifuser will use The same non root user name on every new Linux distro installed by proot-distro, it can happen, that Dbint will coccomplain ant at-spi will refuse to work. There can be even issues with sockets connection. But I have 100 % confirme, that.
Orca can really communicate with Espeak-ng and speech-dispatcher compiled for Termux. I have even removed espeak-ng package from Archlinux to ensure, that it reall communicate with Termux variant of this package.
I have also remove speech-dispatcher module from ArchLinux.
Really, it communicate with Termux variant of those packages.
My dream was to try Alpine Linux. But Marco generates strange error messages related to pixels so I have found out, that ArchLinux or Fedora will be the best.
